frontend/src/components/hooks/useNotificationUsers.js

```diff
@@ -0,0 +1,18 @@
+import { useEffect, useState } from 'react'
+import { getAllNotificationUser } from '../../service/notificationUserService'
+
+export default function useNotificationUsers() {
+  const [notificationUser, setNotificationUser] = useState([])
```

Comment:
notficationUser is maybe a bit confusing. Perhaps userNotification would be more precise